mate-window-picker-applet/task-title.c

```diff
@@ -49,15 +48,66 @@ struct _TaskTitlePrivate
   gboolean mouse_in_close_button;
 };
 
+enum
+{
+  PROP_0,
+  PROP_SHOW_HOME_TITLE
+};
+
 static void disconnect_window (TaskTitle *title);
 
+static gboolean
+start_logout_dialog (TaskTitle *title)
+{
+  GError *error = NULL;
+  GAppInfo *app_info;
+  GdkAppLaunchContext *launch_context;
+  GdkDisplay *display;
+  GdkScreen *gdkscreen;
+  TaskTitlePrivate *priv;
```

Comment:
Build warning.
https://travis-ci.org/mate-desktop/mate-netbook/jobs/521411788#L1175
```
task-title.c:69:21: warning: variable 'priv' set but not used [-Wunused-but-set-variable]
   TaskTitlePrivate *priv;
                     ^~~~
```

Reply:
Oh, sorry about that, it is a forgotten variable. I've fixed it in my last force-push.